packages/components/package.json

```diff
@@ -115,7 +115,7 @@
   },
   "dependencies": {
     "@floating-ui/react-dom": "^2.1.2",
-    "@headlessui/react": "<=1.5.0",
```

Comment:
I am not sure why this was restricted to 1.5.0 or lower. Checked the commit history but didnt see any comments about it